packages/types/package.json

```diff
@@ -18,7 +18,7 @@
         "test": "echo \"Error: run tests from root\" && exit 1"
     },
     "dependencies": {
-        "@cennznet/util": "^1.2.1",
+        "@cennznet/util": "1.3.0-alpha.1",
```

Comment:
This should be ^1.3.0 or remain ^1.2.1 until we have published cennzet/util. Isn't it?

Reply:
The right order of publishing according to the hierarchy of our packages seem to be util -> types -> api, can we enforce that it lerna.json?

Reply:
no.. how monorepo works is.. It will use 1.3.0-alpha.1 from the current packages where packages/util has version as 1.3.0-alpha.1.. Does not have to wait till one package is published and then we can publish the dependent one..
Monorepo will publish all at once..

Reply:
Would ^1.3.0 work as well? If so it would be preferred.